src/utils/list_cpu_features.c

```diff
@@ -340,6 +340,8 @@ static Node* GetCacheTypeString(CacheType cache_type) {
     case CPU_FEATURE_CACHE_PREFETCH:
       return CreateConstantString("prefetch");
   }
+  /* Should never happen */
```

Comment:
Since this should never happen I'd rather use `__builtin_unreachable()` for clang/gcc and `__assume(0)` for MSVC.
Something along these lines:
```
#if defined(CPU_FEATURES_COMPILER_CLANG) || defined(CPU_FEATURES_COMPILER_GCC)
  __builtin_unreachable();
#elif defined(CPU_FEATURES_COMPILER_MSC)
  __assume(0);
#else
#error "Unsupported compiler."
#endif
```

Reply:
ok, done. thanks, I wasn't aware of those builtins.
Instead of issuing an error for compiler other than MSC,GCC,clang I simply do nothing. In this unusual situation, it is a bit better as compilation may still work if something like ``-pedantic`` is not used.